sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/FnService.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.runners.fnexecution;
+package org.apache.beam.sdk.fn;
```

Comment:
Instead of moving classes to `org.apache.beam.sdk.fn`, can we create a new subpackage, such as `org.apache.beam.sdk.fn.server`?

Reply:
Sounds good.

Reply:
+1